src/cli/flags/repo.go

```diff
@@ -17,13 +18,26 @@ const (
 
 var (
 	BackupIDFV           string
+	BackupIDsFV          []string
 	AWSAccessKeyFV       string
 	AWSSecretAccessKeyFV string
 	AWSSessionTokenFV    string
 	CorsoPassphraseFV    string
 	SucceedIfExistsFV    bool
 )
 
+// AddBackupIDFlag adds the --backup flag.
+func AddDeleteBackupIDFlag(cmd *cobra.Command, require bool) {
```

Comment:
Nit: since the flag was changed to not say "delete", and also to make sure it doesn't accidentally get confused with the singular backup.
```suggestion
func AddMultipleBackupIDsFlag(cmd *cobra.Command, require bool) {
```